app/models/graph/schema.rb

```diff
@@ -23,8 +25,5 @@ module Graph
       model = Object.const_get(gid.model_name)
       Graph::FindLoader.for(model).load(gid.model_id.to_i)
     end
-
-    lazy_resolve(Promise, :sync)
-    instrument(:query, GraphQL::Batch::Setup)
```

Comment:
I wonder if I could handle it better in the gem: use a Set instead of an Array to store instrumenters, that way  a second call would do nothing.